packages/server/app/Controllers/Http/Temporal/MtvServerToTemporalController.ts

```diff
@@ -13,7 +13,7 @@ import {
 import got from 'got';
 import urlcat from 'urlcat';
 
-const TEMPORAL_ENDPOINT = Env.get('TEMPORAL_ENDPOINT');
+const MTV_TEMPORAL_ENDPOINT = `${Env.get('TEMPORAL_ENDPOINT')}/mtv`;
```

Comment:
To be sure we encounter no issue if `TEMPORAL_ENDPOINT` environment variable ends with a slash `/` or not, we could use `urlcat` here too.

Reply:
Done